### ISSUE TITLE
Autosuggestions for docker stop working when arguments are passed to docker

### ISSUE THREAD

**Issue by Daniel451** [robbyrussell/oh-my-zsh#8301]
**Is your feature request related to a particular plugin or theme? If so, specify it.**
`docker` plugin

**Is your feature request related to a problem? Please describe.**
Autosuggestions of running containers.

**Describe the solution you'd like**
Commands like `docker exec` + `tab`-key gives autosuggestions of all running containers for command completion. However, if arguments are passed to `docker`, autosuggestions does not recognize the docker command anymore and gives me suggestions of pretty much anything that it could run (`mount`, `ls`, ...).

I suggest something to ignore arguments passed to stuff like `exec` to enable correct autosuggestions for commands like `docker exec -ti`.

**Additional context**
In short:
`docker exec` + `tab`-key gives correct autosuggestions (running docker containers)
`docker exec -ti` + `tab`-key does not give useful autosuggestions, `docker` is not recognzied anymore

**Comment by Daniel451** [comment]
I am terribly sorry, just discovered that my issue is related to https://github.com/robbyrussell/oh-my-zsh/issues/6710
